src/shared/utils/index.js

```diff
@@ -105,9 +125,9 @@ const setChartMargin = (
     left
   ] = useAxis
     ? [
-      TEXT_HEIGHT / 2 + 1,
-      TEXT_HEIGHT / 2 + 1,
-      AXIS_TICK_WIDTH + BUFFER,
+      SYMBOL_SIZE / 2 + 1,
+      SYMBOL_SIZE / 2 + 1,
```

Comment:
_pref:_ `SYMBOL_SIZE` is the size of the legend symbol, so if these aren't related, maybe just use `BUFFER`?

Reply:
For consistency, we shouldn't just add extra space at the beginning around the chart (before extra elements such as tick labels or axes legends) come into view. At the beginning, only symbols stick out on the top and right beyond the actual margin of the chart so we have to make room only for them. The same I am doing at the bottom and left, I only add space for the ticks.

Reply:
Also, I will adjust the margin for pie chart to BUFFER instead of 2*BUFFER as I think it is enough.